packages/lending/src/protocols/morphoblue/lending-protocol.ts

```diff
@@ -135,25 +135,38 @@ export class LendingProtocol extends Protocol {
 
     const maxLtv = common.toBigUnit(lltv, 18);
 
+    const lstTokenAPYMap = await this.getLstTokenAPYMap(this.chainId);
+
+    const supplyApy = '0';
```

Comment:
any reason why we don't use `apy` directly?

Reply:
There's a `borrowApy` within the same scope, using `supplyApy` is fine, but sure we can rename to `apy` for better readability

Reply:
i see. In that case, `supplyApy` is easier to understand. thx.